### PR TITLE
Add a feature list and screenshots to the welcome page

### PR DESCRIPTION
**Pull request by lloydmeta**
Used a few helpers from [the official middleman docs](http://middlemanapp.com/basics/helpers/).

### CODE REVIEW

**Comment by cb372** [comment]
Nice! Maybe we can use screenshots from production (properly censored) to make them slightly more exciting.

File: source/index.html.erb [hunk 1]
```diff
@@ -4,13 +4,30 @@ nav_order: 10
...
+
+  <p>Octoparts is implemented in Scala, making heavy use of the <%= link_to "Hystrix", "https://github.com/Netflix/Hystrix" %> library.</p>
+
 </section>
 
-<p>TODO screenshots</p>
+<section>
+  <h3>Summarised feature list</h3>
+    <ul>
+      <li>Stateless and non-blocking architecture</li>
```
Comment (cb372) [inline source/index.html.erb]: The architecture, while exciting to the people who built it, probably doesn't count as a feature :(

Also I would suggest
- Parallelization of API requests
- Tolerance against backend failure
- Powerful and flexible caching

as features

Comment (lloydmeta) [inline source/index.html.erb]: Ah true. I've added those 3 and removed the first non-feature :)

Comment (lloydmeta) [inline source/index.html.erb]: Good idea re: the screenshots btw. Will do tmw.

**Comment by lloydmeta** [comment]
Thinking about this again; I'm starting to think that being stateless IS a feature that we should mention; certainly it is a lot simpler to scale than a state-ful architecture.

**Comment by cb372** [comment]
OK, go ahead and add it. The more features the merrier!

**Comment by lloydmeta** [comment]
Updated with prod screenies and stateless arch. as a feature.

**Pull request merged**